### PR TITLE
Styled DropdownNativeSelect

### PR DESCRIPTION
**Pull request by guilean**
**Checklist**

- [x] New feature (non-breaking change which adds functionality)

**Please check if the PR fulfills these requirements**

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Storybook or docs have been added / updated (for bug fixes / features)

**What is the ticket / issue associated with this change?** (Link to the ticket or issue)
https://trello.com/c/95Nr1LgN/94-migrate-composer-forms-to-styled-components

### CODE REVIEW

File: packages/styled-components/src/components/primitives/Form/DropdownField/DropdownField.types.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { CssCustomizableProps } from 'components/Components.types';
+
+export interface Option {
```
Comment (tkanzakic) [inline packages/styled-components/src/components/primitives/Form/DropdownField/DropdownField.types.ts:3]: type

**Review by tkanzakic — CHANGES_REQUESTED** [review]

File: packages/styled-components/src/components/primitives/Form/DropdownField/DropdownField.types.ts (hunk 1 above)
Comment (guilean) [reply packages/styled-components/src/components/primitives/Form/DropdownField/DropdownField.types.ts]: Changed

File: packages/styled-components/src/components/primitives/Form/DropdownField/DropdownField.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  title: 'Primitives/Form/DropdownField',
+  component: DropdownField,
+};
+
+const options = [
+  { value: 1, label: 'Option 1' },
+  { value: 2, label: 'Option 2' },
+  { value: 3, label: 'Option 3' },
+];
+
```
Comment (tkanzakic) [inline packages/styled-components/src/components/primitives/Form/DropdownField/DropdownField.stories.tsx:16]: please add a story with the placeholder and another one with an item selected

File: packages/styled-components/src/components/primitives/Form/DropdownField/DropdownField.tsx [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+export const DropdownField = forwardRef<HTMLSelectElement, Props>(
+  (props: Props, ref) =>
+    isMobile ? (
+      <DropdownMobile {...props} ref={ref} />
+    ) : (
+      <DropdownDesktop {...props} ref={ref} />
+    )
+);
+
+export const DropdownMobile = forwardRef<HTMLSelectElement, Props>(
```
Comment (tkanzakic) [inline packages/styled-components/src/components/primitives/Form/DropdownField/DropdownField.tsx:32]: we should not export this

File: packages/styled-components/src/components/primitives/Form/DropdownField/DropdownField.tsx [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+          <StyledIconMobile>
+            <ExpandMore type={Icon.Types.Filled} />
+          </StyledIconMobile>
+        </StyledIconWrapperMobile>
+      </StyledWrapperMobile>
+    );
+  }
+);
+
+export const DropdownDesktop = forwardRef<HTMLSelectElement, Props>(
```
Comment (tkanzakic) [inline packages/styled-components/src/components/primitives/Form/DropdownField/DropdownField.tsx:87]: ditto

**Review by tkanzakic — CHANGES_REQUESTED** [review]

File: packages/styled-components/src/components/primitives/Form/DropdownField/DropdownField.stories.tsx (hunk 2 above)
Comment (guilean) [reply packages/styled-components/src/components/primitives/Form/DropdownField/DropdownField.stories.tsx:16]: Done

File: packages/styled-components/src/components/primitives/Form/DropdownField/DropdownField.tsx (hunk 3 above)
Comment (guilean) [reply packages/styled-components/src/components/primitives/Form/DropdownField/DropdownField.tsx]: Removed

File: packages/styled-components/src/components/primitives/Form/DropdownField/DropdownField.tsx (hunk 4 above)
Comment (guilean) [reply packages/styled-components/src/components/primitives/Form/DropdownField/DropdownField.tsx]: Removed

**Review by tkanzakic — APPROVED** [review]

**Pull request merged**